app/elements/app-user-menu/app-user-menu.html

```diff
@@ -3,131 +3,137 @@
 <link rel="import" href="../../bower_components/paper-menu/paper-menu.html">
 <link rel="import" href="../../bower_components/paper-item/paper-item.html">
 <link rel="import" href="../../bower_components/paper-menu-button/paper-menu-button.html">
+<link rel="import" href="../organizations/organization-add.html">
```

Comment:
Is it a good idea to preload the add org form in every user menu? Maybe we should only load it after the user clicks on the create org button.